scripts/icon_studio/controllers/icon_creator/objects.json

```diff
@@ -6,962 +6,1680 @@
   },
   "projects/objects/traffic/protos/TrafficLightBigPole": {
-    "fields": "rotation 0 1 0 1.0854"
+    "fields": "  rotation 0.0 0.0 1.0 1.0854",
+    "nodeString": "  TrafficLightBigPole{ rotation 0.0 0.0 1.0 1.0854 }"
   },
   "projects/objects/traffic/protos/TrafficLightHorizontal": {
-    "nodeString": "Robot { rotation 0 1 0 5.23 children [ TrafficLightHorizontal { } ] }"
+    "fields": "  rotation 0.0 0.0 1.0 5.23",
+    "nodeString": "  Robot { rotation 0.0 0.0 1.0 5.23 children [ TrafficLightHorizontal { } ] }"
   },
   "projects/objects/traffic/protos/TrafficLightStandardLampGeometry": {
-    "nodeString": "Transform { translation 0 0.1 0 rotation 1 0 0 1.57 children [ Shape { appearance PBRAppearance { baseColor 1 0 0 metalness 0 roughness 0.9 } geometry TrafficLightStandardLampGeometry { } } ] }"
-  },
-  "projects/objects/traffic/protos/TrafficLightArrowLampGeometry": {
-    "nodeString": "Transform { translation 0 -0.019 0 rotation 1 0 0 1.57 children [ Shape { appearance PBRAppearance { baseColor 1 0 0 metalness 0 roughness 0.9 } geometry TrafficLightArrowLampGeometry { } } ] }"
+    "fields": "  rotation 0.0 -1.0 0.0 1.57",
+    "nodeString": "  Transform { translation 0 0 0.1 rotation 0.0 -1.0 0.0 1.57 children [ Shape { appearance PBRAppearance { baseColor 1 0 0 metalness 0 roughness 0.9 } geometry TrafficLightStandardLampGeometry { } } ] }"
   },
   "projects/objects/traffic/protos/WorkBarrier": {
-    "fields": "rotation 0 1 0 1.3472"
+    "fields": "  rotation 0.0 0.0 1.0 1.3472",
+    "nodeString": "  WorkBarrier{ rotation 0.0 0.0 1.0 1.3472 }"
   },
   "projects/objects/traffic/protos/YieldPanel": {
-    "background": [0.0, 1.0, 1.0],
-    "fields": "translation 0 0.48 0 rotation 0 1 0 4.75"
+    "background": [
+      0.0,
+      1.0,
+      1.0
+    ],
+    "fields": "  translation 0 0 0.48 rotation 0.0 0.0 1.0 4.75",
+    "nodeString": "  YieldPanel{ translation 0 0 0.48 rotation 0.0 0.0 1.0 4.75 }"
   },
   "projects/objects/traffic/protos/YieldSign": {
-    "background": [0.0, 1.0, 1.0],
-    "fields": "rotation 0 1 0 4.75059"
+    "background": [
+      0.0,
+      1.0,
+      1.0
+    ],
+    "fields": "  rotation 0.0 0.0 1.0 4.75059",
+    "nodeString": "  YieldSign{ rotation 0.0 0.0 1.0 4.75059 }"
   },
   "projects/objects/trees/protos/BigSassafras": {
-    "background": [1.0, 1.0, 1.0]
+    "background": [
+      1.0,
+      1.0,
+      1.0
+    ],
+    "fields": "  rotation 0.0 0.0 1.0 0.3",
+    "nodeString": "  BigSassafras{ rotation 0.0 0.0 1.0 0.3 }"
   },
   "projects/objects/trees/protos/Cypress": {
-    "background": [1.0, 1.0, 1.0]
+    "background": [
+      1.0,
+      1.0,
+      1.0
+    ],
+    "fields": "  rotation 0.0 0.0 1.0 0.3",
+    "nodeString": "  Cypress{ rotation 0.0 0.0 1.0 0.3 }"
   },
   "projects/objects/trees/protos/Forest": {
-    "background": [1.0, 1.0, 1.0],
-    "fields": "translation 0 0.1 0 rotation 0 1 0 0.3"
+    "background": [
```

Comment:
Forest has a bug, likely an issue with the PROTO itself.
![a](https://user-images.githubusercontent.com/44834743/139904647-05d6709d-fe6e-4a48-91b3-8ee1be1a24db.png)